### ISSUE TITLE
Low correlation in GFP dataset

### ISSUE THREAD

**Issue by charlesxu90** [chloechsu/combining-evolutionary-and-assay-labelled-data#2]
Dear author,

Thanks for your excellent work. It's a good point for further studies. I'm working on GFP case. I tried your code on GFP, it gives a spearman correlation score of 0.5510 on the original dataset you provided with 80/20 split. I think it is smaller than what you presented on your paper, which is around 0.7.

However, when I'm trying with your `ev+onehot` model on GFP datasets, such as ‘Sarkisyan’ (0.107), ‘SynNeigh’  (0.026) or ‘FP Homologs’ (0.011), it gives me a very poor spearman correlation. 

Here, I'm trying to use as much dataset as possible. But seems the low N-train models are not working on cases with large number of sequences. Have you ever tried that? Do you have any ideas how this come? 

Is it possible for us to make the same model working for both cases, i.e. with few training samples (Low-N), and thousands of training samples?

**Comment by charlesxu90** [comment]
In my own opinion, if a model is not working well on large dataset, it should not be able to give a reliable prediction with small training samples. I'm not quiet sure about the reason, but I'll double check with my experiments.

**Comment by mingchen-li** [comment]
That's true! I have tried the code and got the same conclusion with yours.
